src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -634,6 +684,8 @@ public static void onWorldLoad(@NotNull final World world)
         {
             if (numWorldsLoaded == 0)
             {
+                backupColonyData();
```

Comment:
the result is ignored. We should print out something, depending on if it worked or not